app/views/miq_ae_customization/_dialog_field_form.html.haml

```diff
@@ -73,8 +74,17 @@
                           @edit[:field_trigger_auto_refresh],
                           'data-miq_observe_checkbox' => {:url => url}.to_json)
 
-  - if @edit[:field_typ] =~ /Drop|Radio/ && !@edit[:field_dynamic]
-    = render :partial => 'field_values', :locals => {:entry => nil}
+  - if @edit[:field_typ] =~ /Drop|Radio/
```

Comment:
Won't this make it so that the Multiselect option will appear for Radio button type dialogs too?